x-pack/legacy/plugins/canvas/public/components/autocomplete/autocomplete.scss

```diff
@@ -13,10 +13,11 @@
   border-right: $euiBorderThin;
 }
 
-.autocompleteItems, .autocompleteReference {
+.autocompleteItems,
+.autocompleteReference {
+  @include euiScrollBar;
   height: 258px;
```

Comment:
Did it not yell about the pixel value here?

Reply:
Nope, it's not giving any errors for px values. Maybe it needs to be configured or something?

Reply:
I thought for sure I set this up. I guess not. We don't have it on in EUI either. Something we can look into in a later PR.

https://github.com/sasstools/sass-lint/blob/master/docs/rules/variable-for-property.md